foreman.te

```diff
@@ -227,14 +205,6 @@ optional_policy(`
 
 optional_policy(`
     tunable_policy(`passenger_run_foreman', `
-        read_files_pattern(passenger_t, httpd_foreman_script_exec_t, httpd_foreman_script_exec_t)
-        read_lnk_files_pattern(passenger_t, httpd_foreman_script_exec_t, httpd_foreman_script_exec_t)
-        manage_files_pattern(passenger_t, foreman_log_t , foreman_log_t)
```

Comment:
No, this line only as it's unrelated to the httpd_* types.  Passenger still needs management of the foreman log files.